docs/defects-api.yml

```diff
@@ -324,7 +295,10 @@ components:
         inspectionTypes:
           type: array
           items:
-            $ref: "#/components/schemas/inspectionType"
+            type: string
```

Comment:
can this not still reference the type? if not, can the type be removed?

Reply:
Shout, yes it can